lightning/src/ln/channelmanager.rs

```diff
@@ -131,7 +131,7 @@ pub use crate::ln::outbound_payment::{Bolt12PaymentError, ProbeSendFailure, Retr
 #[cfg(test)]
 pub(crate) use crate::ln::outbound_payment::PaymentSendFailure;
 use crate::ln::script::ShutdownScript;
-
+use crate::routing::gossip::NodeId;
```

Comment:
nit: move this up to the other `routing` import and keep the newline between the imports and the start of the code